app/models/dto.py

```
@@ -67,6 +67,75 @@ class AvailabilityRequest(BaseModel):
     neLat: float = Field(..., description="North-East Latitude")
     neLng: float = Field(..., description="North-East Longitude")
 
+    @field_validator('date')
+    @classmethod
+    def validate_date_format(cls, v: str) -> str:
+        import re
+        from datetime import datetime, date
+        
+        # 1. Format Check
+        if not re.match(r"^\d{4}-\d{2}-\d{2}$", v):
+            raise ValueError("날짜 형식이 올바르지 않습니다. (YYYY-MM-DD)")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)